filter/etc/rspamd/scores.d/nethserver_group.conf

```diff
@@ -1,11 +1,14 @@
 #
 # This file will be overwritten by the next rpm update
 # use /etc/rspamd/local.d/nethserver_group.conf' - to override the defaults
-#
+# 
 
 symbols = {
     "IP_REPUTATION" {
-        score = 0.0;
+        weight = 0.0;
```

Comment:
Why this change?

Reply:
https://rspamd.com/doc/configuration/metrics.html#symbols

do not know, vlad suck or delabrusse suck :)

It is workable with score, but doc now states weight